frontend/app/src/components/inputs/AssetSelect.vue

```diff
@@ -61,6 +65,17 @@
         {{ tc('asset_select.no_results') }}
       </div>
     </template>
+    <template #append>
+      <div v-if="loading" class="fill-height d-flex items-center">
```

Comment:
Does this replace autocomplete loading?

Reply:
not replacing, but showing it in more visible place. By default the progress bar is located on the bottom, but it is blocked by the list items.

Reply:
I am aware, the question is since we are adding this visible one, wouldn't it make sense for us to remove the `:loading` property from the autocomplete?